projects/storefrontlib/src/cms-components/user/checkout-login/checkout-login.component.ts

```diff
@@ -46,7 +46,10 @@ export class CheckoutLoginComponent implements OnDestroy {
 
```

Comment:
good good, I can approve tomorrow morning after talking!
Also, I think in the form control for email confirmation, the custom validation is redundant, because email must match anyway. I think its better to remove it.
Lastly, should we align the message to keep conciseness with the update email form or not in terms of validations